mypy.ini

```diff
@@ -5,9 +5,10 @@ files = src/evaluation_system/model/*.py,
     src/evaluation_system/api
 strict = False
 
-warn_unused_ignores = True
+warn_unused_ignores = False
```

Comment:
Why did you do that?

Reply:
`mypy-precommit` and `mypy` don't have the same behavior. I was just getting this errors in `mypy-precommit` and not in the newest version of `mypy`. To fast fixing of two `mypy` errors I made it False. Cause there were two `# type: ignore` and they needed to have a deep look at the types in source code and still I'm reading the code. Long story short so I didn't want to touch the code a lot before getting deep knowledge.